src/main/src/main/java/org/geoserver/catalog/CoverageDimensionCustomizerReader.java

```diff
@@ -237,8 +237,12 @@ public GridCoverage2D read(String coverageName, GeneralParameterValue[] paramete
                     dimensions = new ArrayList<>();
                     List<CoverageBand> coverageBands = coverageView.getCoverageBands();
                     for (CoverageBand band : coverageBands) {
+                        CoverageView.CompositionType compositionType = band.getCompositionType();
```

Comment:
By looking at the UI, I thought the cover view had a composition mode, but here it's on the band... I see it was in the original design, still, looks odd.

Reply:
It's not used anymore.